manuscript/03.results.md

```diff
@@ -36,7 +36,7 @@ To evaluate how accurately a tHMM model could infer the behavior of multi-state
 
-![**Model performance on censored lineages of increasing breadth and depth.** (a) Two-state populations of increasing breadth (increasing number of initial cells and therefore lineages) and of increasing depth (increasing experiment time) are analyzed. The states are shown in green and blue colors and red shows cell death. Each point in the scatter plots represents the estimated value for a lineage with the number of cells shown in the x-axis. The solid lines are the Lowess lines for the scatter points. In (b-c) it shows the accuracy in estimating the Bernoulli parameters for G1 and S/G2 phase, respectively, and (d) depicts the distance between the true and estimated Gamma distributions associated with phase lengths for the two states. The model performance is shown based on (e) state assignment accuracy, (f) the error between the estimated and the true transition rate matrix and the (g) initial probability vector. Note that the Wasserstein distance between the true and estimated distributions for each state is much lower than the distance for two distributions that are quite similar (Fig. {@fig:wass}b).](figure5.svg){#fig:performSyn}
+![**Model performance on censored lineages of increasing breadth and depth.** (a) Two-state populations of increasing breadth (increasing number of initial cells and therefore lineages) and of increasing depth (increasing experiment time) are analyzed. The states are shown in green and blue colors and red shows cell death. Each point in the scatter plots represents the estimated value for a lineage with the number of cells shown in the x-axis. The solid lines are the Lowess lines for the scatter points. The accuracy in estimating the Bernoulli parameters, which imply cell death or cell division, for G1 and S/G2 phase, respectively, is shown in (b-c). The distance between the true and estimated Gamma distributions associated with phase lengths for the two states is depicted in (d). This is measured using the Wasserstein distance. The model performance shown is based on state assignment accuracy (e), the error between the estimated and the true transition rate matrix (f), and the initial probability vector (g). Note that the Wasserstein distance between the true and estimated distributions for each state is much lower than the distance for two distributions that are quite similar (Fig. {@fig:wass}b).](figure5.svg){#fig:performSyn}
```

Comment:
The explanations you added should go to the main text, not the caption.

Reply:
Thank you for letting me know! I just made some changes in my most recent branch. Please let me know what you think. I had also repositioned the cartoon for Supplementary Figure 9 and 10 to be a bit more centered

Reply:
Thanks! Sounds good to me. Please ask aarmey for merge.